resources/developer-guides/daemon-rpc.md

```diff
@@ -240,19 +250,29 @@ Outputs:
 
 * *block_header* - A structure containing block header information.
   * *block_size* - unsigned int; The block size in bytes.
+  * *block_weight* - TODO
+  * *cumulative_difficulty* - unsigned int; Cumulative difficulty of all blocks in the blockchain.
```

Comment:
Only up to the block in the reply

Reply:
Changing to `unsigned int; Cumulative difficulty of all blocks up to the block in the reply.`